solutions/21.js

```diff
@@ -0,0 +1,36 @@
+// Rahul 
+// Find the first three largest numbers 
+// input: [8,1,4,2,12,6,7,19,2,9]
+// output: 9,12,19 
+let array = [ 1,1,2,3,4,3,4];
+
+const solution = (array) => {
+  max =0; newArray = [];
+  
+
+  for(i=0; i<array.length-1; i++)
+  {
+    if(array[i] >= array[i+1])
+    {
+      max = array[i];
+      array[i] = array[i+1];
+      array[i+1] = max;
+  }
+  }
+  console.log('array:',  array);
+
+  for(j=array.length-1; j>=0; j--)
+  {
+    newArray.push(array[j]);
+  }
+  
+  console.log('newArray:', newArray);
+  console.log('First Three Large Numbers:', newArray[0],newArray[1], newArray[2]);
+ 
+};
+
+module.exports = {
+  solution
+}
+
+solution(array);
```

Comment:
@rkalra247  please remove the following because you will not be calling the solution function on this file.